src/Symfony/Bridge/Monolog/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^7.2.9",
         "monolog/monolog": "~1.19",
         "symfony/service-contracts": "^1.1",
-        "symfony/http-kernel": "^4.4|^5.0"
+        "symfony/http-kernel": "^5.0"
```

Comment:
for my information, why is this necessary? shouldn't using the  new event classes also work in 4.4?

Reply:
I don't think this is necessary will revert

Reply:
this uneccessary dependency raise happend in several places